benchmark/utils/generator.hpp

```diff
@@ -86,16 +119,22 @@ struct DefaultSystemGenerator {
 
     static std::string describe_config(const json& config)
     {
+        std::string raw_desc;
         if (config.contains("filename")) {
-            return config["filename"].get<std::string>();
+            raw_desc = config["filename"].get<std::string>();
         } else if (config.contains("stencil")) {
             std::stringstream ss;
             ss << "stencil(" << config["size"].get<gko::int64>() << ", "
                << config["stencil"].get<std::string>() << ")";
-            return ss.str();
+            raw_desc = ss.str();
         } else {
             throw std::runtime_error("No known way to describe config.");
         }
+        if (config.contains("reorder")) {
+            auto reorder_name = config["reorder"].get<std::string>();
+            raw_desc = reorder_name + "(" + raw_desc + ")";
```

Comment:
That might just be my perception, but I think this puts too much emphasis on the reorder. Maybe `matrix_desc(reorder)` would be better.